VAMobile/src/screens/HealthScreen/Appointments/AppointmentDetailsCommon/TypeOfCare.test.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import { screen } from '@testing-library/react-native'
+
+import { context, render } from 'testUtils'
+import TypeOfCare from './TypeOfCare'
+import { defaultAppointmentAttributes } from 'utils/tests/appointments'
+
+context('TypeOfCare', () => {
+  let props: any
```

Comment:
Can define this inside the function instead

Reply:
updated